streams/src/main/java/org/apache/kafka/streams/kstream/KGroupedStream.java

```diff
@@ -39,25 +39,27 @@
      * {@link KTable}.
      *
      * @param reducer           the instance of {@link Reducer}
-     * @param name              the name of the resulted {@link KTable}
+     * @param storeName         the name of the underlying {@link KTable} state store
```

Comment:
Let's add comments explaining about the store names as well as the changelog names for all the functions that require this parameter. For example taking this function:

```
Combine values of this stream by the grouped key into a new instance of ever-updating {@link KTable}.
The resulted {@link KTable} will be materialized in a local state sore associated the given store name. Also a changelog topic named "applicationID-storeName-changelog" will be automatically created in Kafka for replication, where "applicationID" is user-specified in the {@code StreamsConfig}.
```